src/Refactoring-Core/RBDirectlyDefinesInstanceVariableCondition.class.st

```diff
@@ -35,3 +35,14 @@ RBDirectlyDefinesInstanceVariableCondition >> instanceVariables: aColOfStrings [
 
 	instanceVariables := aColOfStrings
 ]
+
+{ #category : 'displaying' }
+RBDirectlyDefinesInstanceVariableCondition >> violationMessageOn: aStream [
```

Comment:
I think we should think about pulling this method up to RBNewAbstractCondition, like below

```
RBNewAbstractCondition 
    self subclassResponsibility
```